reconcile/utils/gql.py

```diff
@@ -218,7 +231,28 @@ def commit_timestamp_utc(self) -> Optional[str]:
         return None
 
 
-_gqlapi: Optional[GqlApi] = None
+class GqlApiSingleton:
+    gql_api: Optional[GqlApi] = None
+    gqlapi_lock = threading.Lock()
+
+    def __new__(cls, *args, **kwargs):
+        with cls.gqlapi_lock:
+            if not cls.gql_api:
+                cls.gql_api = GqlApi(*args, **kwargs)
+        return cls
+
+    @classmethod
+    def instance(cls):
+        if not cls.gql_api:
+            raise GqlApiError("gql module has not been initialized.")
+        return cls
```

Comment:
here we can return instance `cls.gql_api` to match method name

Reply:
Will adjust